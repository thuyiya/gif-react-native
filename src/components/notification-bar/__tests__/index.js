/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import { NotificationBar } from "_components";
import { Provider } from "_contexts";

describe("NotificationBar", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Provider value={{ state: { notification: {} } }}>
        <NotificationBar />
      </Provider>
    );
  });

  it("enders correctly react-test-renderer", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
