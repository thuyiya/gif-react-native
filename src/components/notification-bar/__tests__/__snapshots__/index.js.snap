// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotificationBar enders correctly react-test-renderer 1`] = `ShallowWrapper {}`;
